.circleci/run_e2e.sh

```diff
@@ -3,11 +3,24 @@
 CNI_HUB=${CNI_HUB:-istio}
 CNI_TAG=${CNI_TAG:-$1}
 
+chartdir=$(pwd)/charts
+mkdir ${chartdir}
+
 cd /go/src/istio.io/istio
 
 HUB=${HUB:-gcr.io/istio-release}
 TAG=${TAG:-master-latest-daily}
 
+helm init --client-only
+helm repo add istio.io https://storage.googleapis.com/istio-release/releases/1.1.0-snapshot.6/charts
+helm fetch --untar --untardir ${chartdir} istio.io/istio-cni
+
+helm template --values ${chartdir}/istio-cni/values.yaml --name=istio-cni --namespace=istio-system --set "excludeNamespaces={}" --set hub=${CNI_HUB} --set tag=${CNI_TAG} --set pullPolicy=IfNotPresent --set logLevel=${CNI_LOGLVL:-debug}  ${chartdir}/istio-cni > istio-cni_install.yaml
+
+kubectl create ns istio-system
+kubectl apply -f istio-cni_install.yaml
+
+
 HUB=${HUB} TAG=${TAG} make istioctl
 
 # Remove any pre-existing charts
```

Comment:
You should remove all istio-cni.X helm settings since the istio-cni chart was removed in 1.1 and presumably will eventually be removed from master.   Also since you are installing cni above if either master or the release still has istio-cni integrated things are likely to break.   Note also as best I can tell master has not removed the subchart so that is going to make it difficult to test.

Reply:
yeah... the extra helm settings for istio-cni.X will all change.  They don't hurt so I didn't touch the line.  This file diff is WIP for now.